avm/res/insights/scheduled-query-rule/main.bicep

```diff
@@ -116,11 +122,12 @@ resource queryRule 'Microsoft.Insights/scheduledQueryRules@2021-02-01-preview' =
     autoMitigate: (kind == 'LogAlert') ? autoMitigate : null
     criteria: criterias
     description: alertDescription
-    displayName: name
+    displayName: !empty(alertDisplayName) ? alertDisplayName : name
```

Comment:
```suggestion
    displayName: alertDisplayName ?? name
```

Reply:
overall looks great and you're of course invited to push back as it's just a minor suggest I made. If you choose to use them though, please re-run the `Set-AVMModule` script after so that the `main.json` file is updated accordingly (: